frigate/data_processing/common/license_plate/mixin.py

```diff
@@ -1308,12 +1395,23 @@ def lpr_process(
             None,
         )
 
-        # If it's a known plate, publish to sub_label
-        if sub_label is not None:
+        # don't overwrite sub label for objects that have a sub label
+        # that is not a license plate
+        if (
+            sub_label is not None
+            and obj_data.get("sub_label") is not None
+            and id not in self.detected_license_plates
```

Comment:
I think we will need to guard where detected_license_plates is set too

Reply:
In what way?

Reply:
We aren't guarding it right now so the logic is not the same. If a license plate is recognized and known it will get added to known plates and that means FedEx for example will be overwritten if the plate is a known plate. 

That might be intentional but it's not working the same way as before, since this check now happens after detected_license_plates may be set

Reply:
Makes sense - I will just remove the check for `and id not in self.detected_license_plates` so that any existing sub labels will never get overwritten.

Reply:
Though I suppose I could see a case for a sub label to be overwritten in this case if your spouse is the driver of the same fedex truck and you'd want to know when she got home :)

Reply:
Yeah, mostly just worried about false positives but with plates seems less likely so maybe fine to remove this altogether